src/cloudWatchLogs/commands/searchLogGroup.ts

```diff
@@ -98,10 +116,13 @@ export interface SearchLogGroupWizardResponse {
 }
 
 export class SearchLogGroupWizard extends Wizard<SearchLogGroupWizardResponse> {
-    public constructor(logGroups: string[]) {
+    public constructor(logGroups: string[], logGroupInfo?: CloudWatchLogsGroupInfo) {
         super()
-
-        this.form.logGroup.bindPrompter(() => createLogGroupPrompter(logGroups))
+        if (!logGroupInfo) {
+            this.form.logGroup.bindPrompter(() => createLogGroupPrompter(logGroups))
+        } else {
+            this.form.logGroup.setDefault(logGroupInfo.groupName)
+        }
```

Comment:
This is a good approach, just want to add an alternative:
```ts
super({ initState: { logGroup: logGroupInfo?.groupName } })
```

Then do this:
```suggestion
        this.form.logGroup.bindPrompter(() => createLogGroupPrompter(logGroups))
```

This is because the wizard won't prompt if it starts with what it needs. Less branches are _usually_ a good thing although it comes at the cost of flexibility. 

We should also add a test case for the wizard about the expected flow when the wizard is given different `logGroupInfo`.

Reply:
I tried to adapt this to the new wizard I merged in. Let me know if I missed something.